cedar-policy-cli/CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
 - Requests are now validated by default if a schema is provided. This can be
   disabled with `--request-validation=false`.
 - The `-s` short form can now be used for `--schema` across all subcommands.
+- The `-p`/`--policies` flag can now be omitted across all subcommands where it
+  is present. If the flag is omitted, policies will be read from `stdin`.
+- `--policy-format` flag to many subcommands, allowing you to pass policies in
```

Comment:
```suggestion
- Add `--policy-format` flag to many subcommands, allowing you to pass policies in
```

Reply:
This is in the `Added` section, so I thought we don't do this?  Although it seems we're more careful about that in the `cedar-policy` changelog than this CLI one